flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+  "only-arches": [
+      "x86_64"
+  ],
+  "automerge-flathubbot-prs": true
```

Comment:
Please don't automerge on applications, this means builds will get pushed to users without any testing